src/android/InAppBrowser.java

```diff
@@ -131,14 +134,17 @@ Licensed to the Apache Software Foundation (ASF) under one
             FOOTER_COLOR,
             CUSTOM_TOOLBAR_TITLE,
             CUSTOM_TOOLBAR_BACKGROUND_COLOR,
-            CUSTOM_TOOLBAR_FOREGROUND_COLOR
+            CUSTOM_TOOLBAR_FOREGROUND_COLOR,
+            CUSTOM_LOADER
     );
 
     private final static int DEFAULT_BACKGROUND_COLOR = 0xffffffff;
     private final static int DEFAULT_FOREGROUND_COLOR = 0xff000000;
+    private final static String DEFAULT_LOADER = InAppBrowserViewUtil.LOADER_NONE;
```

Comment:
🎗to decide on whether this should be `default` instead since iOS has this behaviour to maintain backwards compatibility

Reply:
Will update to match iOS

Reply:
Updated